R/make_plots.R

```diff
@@ -5,122 +5,160 @@ library(scales)
 library(reshape2)
 library(stringi)
 
-#' Line chart of revenue over time.
-#'
-#' \code{plot_revenue_over_time} returns a line chart showing revenue over 
-#' time for both baseline and hypothetical rate structures. The baseline
-#' rate is shown in black and the hypthotical in blue.
-#'
-#' @param data The dataframe filtered by date range and rate code.
-#' 
-#' @return A plotly object created from a ggplot chart, with plotly's
-#' modebar removed.
-plot_revenue_over_time <- function(data){
+#******************************************************************
+# Line chart showing revenue over time for both baseline
+# and hypothetical rate structures
+#******************************************************************
+plot_revenue_over_time <- function(data, display_type){
```

Comment:
Looks like somehow the extended comments got overwritten? Maybe the merge got messed up.